packages/contracts/src/make-genesis.ts

```diff
@@ -89,5 +106,30 @@ export const makeStateDump = async (cfg: RollupDeployConfig): Promise<any> => {
     }
   }
 
-  return dump
+  return {
+    config: {
+      chainId: cfg.l2ChainId,
+      homesteadBlock: 0,
+      eip150Block: 0,
+      eip155Block: 0,
+      eip158Block: 0,
+      byzantiumBlock: 0,
+      constantinopleBlock: 0,
+      petersburgBlock: 0,
+      istanbulBlock: 0,
+      muirGlacierBlock: 0,
+      clique: {
+        period: 0,
+        epoch: 30000,
```

Comment:
Are you sure that this is the correct epoch?

Reply:
Yes, it's the same epoch that was previously being used, see `params.AllCliqueProtocolChanges`.